editor/editor_file_system.h

```diff
@@ -189,6 +190,7 @@ class EditorFileSystem : public Node {
 	bool revalidate_import_files = false;
 	int nb_files_total = 0;
 
+	void _update_filesystem();
```

Comment:
I'd call it `_filesystem_changed()`.

Reply:
Or `_notify_filesystem_changed` to be even more clear (`_filesystem_changed` implies it's a handler *connected* to such a signal)